README.md

```diff
@@ -1 +1,189 @@
-# capstone-ebike-reservation
\ No newline at end of file
+<a name="readme-top"></a>
+
+<div align="center">
+
+  <h1><b>Ebike Reservation BackEnd</b></h1>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+## 📗 Table of Contents
```

Comment:
- [ ] Kindly, include in your instructions the required db commands that your users will need to run your backend app: ``rails db:create, rails db:migrate``. 👍